core/src/OpenMP/Kokkos_OpenMP_Team.hpp

```diff
@@ -91,6 +93,9 @@ class TeamPolicyInternal<Kokkos::OpenMP, Properties...>
     int max_host_team_size = Impl::HostThreadTeamData::max_team_members;
     return pool_size < max_host_team_size ? pool_size : max_host_team_size;
   }
+
+  int vector_length() const { return 1; }
```

Comment:
I know CUDA has this function, but I think it is a mistake in the public interface. Since it doesn't seem to exist for all backends and thus shouldn't really work portable I think we could argue to fix this as a "bug in our implementation interface" and make it impl_vector_length for all of them ...

Reply:
Yeah, I don't want to change the existing interface of TeamPolicy in a Tuning PR